src/node_crypto.cc

```diff
@@ -5682,6 +5684,13 @@ bool DiffieHellman::Init(const char* p, int p_len, const char* g, int g_len) {
   return VerifyContext();
 }
 
+inline const modp_group* FindDiffieHellmanGroup(const char* name) {
+  for (const modp_group& group : modp_groups) {
```

Comment:
are the supported group names described somewhere?

Reply:
[Yes](https://nodejs.org/api/crypto.html#crypto_crypto_getdiffiehellman_groupname). I put a link to that section to the new API.